requirements.txt

```diff
@@ -1 +1,10 @@
--e .
```

Comment:
I think we want to keep the setup.py from develop and add the dash dependencies to the setup.py

Reply:
done